online_catalog/TRLN-Discovery/ICPSR/forge-icpsr-codebook2-0-draft.xsl

```diff
@@ -75,13 +75,13 @@
     "id": "<xsl:text>UNCICPSR</xsl:text><xsl:value-of select="$ICPSR-id"/>",
     "rollup_id": "<xsl:text>ICPSR</xsl:text><xsl:value-of select="$ICPSR-id"/>",
     "record_data_source": [
-        "ICPSR"
+        "DDI", "Shared Records", "ICPSR"
     ],
     "url": [
-        "{\"href\":\"<xsl:value-of select="$host"/><xsl:value-of select="$ICPSR-id"/>\",\"text\":\"<xsl:text>Access restricted ; authentication may be required.</xsl:text>\"}",
-        "{\"href\":\"<xsl:text>http://www.icpsr.umich.edu/icpsrweb/ICPSR/help/\</xsl:text>",<xsl:text>ICPSR help for Duke users</xsl:text>\"}",
-        "{\"href\":\"<xsl:text>http://www.lib.ncsu.edu/data/icpsr.html</xsl:text>\",\"text\":\"<xsl:text>ICPSR help for NCSU users</xsl:text>\"}",
-        "{\"href\":\"<xsl:text>http://guides.lib.unc.edu/aecontent.php?pid=455857</xsl:text>\",\"text\":\"<xsl:text>ICPSR help for UNC users</xsl:text>\"}"
+        "{\"type\":\"fulltext\",\"href\":\"<xsl:value-of select="$host"/><xsl:value-of select="$ICPSR-id"/>\",\"text\":\"<xsl:text>Access restricted ; authentication may be required.</xsl:text>\"}",
```

Comment:
Added URL type elements. I don't think the Argot url field was documented well until after you worked on this, but it has been updated, explaining the different types: https://github.com/trln/data-documentation/blob/master/argot/spec_docs/url.adoc